File: app/src/main/java/in/mvpstarter/sample/ui/main/MainMvpView.java
package in.mvpstarter.sample.ui.main;

import java.util.List;

import in.mvpstarter.sample.ui.base.MvpView;

public interface MainMvpView extends MvpView {

    void showPokemon(List<String> pokemon);

    void showProgress(boolean show);

    void showError(Throwable error);

}

File: app/src/main/java/in/mvpstarter/sample/ui/main/MainMvpView.kt
package `in`.mvpstarter.sample.ui.main

import `in`.mvpstarter.sample.ui.base.MvpView

interface MainMvpView : MvpView {

    fun showPokemon(pokemon: List<String>)

    fun showProgress(show: Boolean)

    fun showError(error: Throwable)

}